exercise/views.py

```diff
@@ -148,33 +149,30 @@ def post(self, request, *args, **kwargs):
                 new_submission = Submission.objects.create_from_post(
                     self.exercise, students, request)
             except ValueError as error:
-                messages.error(
-                    request,
-                       _("The submission could not be saved for some reason. " 
-                         "The submission was not registered. Error: {}").format(error)
+                messages.error(request,
+                    _("The submission could not be saved for some reason. "
```

Comment:
This could use a different string "due to malformed post data" or such to indicate it's the users fault.